### ISSUE TITLE
Issue with react version

### ISSUE THREAD

**Issue by Ishaan28malik** [wrakky/react-html-parser#105]
Facing an issue related to `react-18` and `react-16`

![Screenshot from 2023-08-09 17-26-13](https://github.com/peternewnham/react-html-parser/assets/27343592/88ad7a04-20b8-41e1-8b5a-9c8a2341db3f)

**Comment by Ishaan28malik** [comment]
@peternewnham @kevinzwhuang @mjj2000

**Comment by aliashrafi051** [comment]
I have this error too

**Comment by singodiyashubham87** [comment]
@Ishaan28malik @aliashrafi051  I'm also facing this issue, maybe the latest react version doesn't support this library and this library is also not updated and configured for the latest react version.

**Comment by KellsCodes** [comment]
I ran into this issue now, have anyone been able to solve it?

**Comment by singodiyashubham87** [comment]
@KellsCodes Nope! The library is still not updated and therefore not supported by the latest version of react.

**Comment by abdullah43577** [comment]
yeah, I noticed this same issue while I tried using it today

**Comment by lukahukur** [comment]
nobody cares about this library and more and this is why you should not use unnecessary dependency for easy tasks like this.

**Comment by Gnomepman** [comment]
I have the same one

**Comment by robhadfield** [comment]
oh no! Came here to post the same. Anyone found a workaround yet?

**Comment by vladsovetov** [comment]
We just replaced this lib with https://www.npmjs.com/package/html-react-parser with no code change

**Comment by deepakraj5** [comment]
> We just replaced this lib with https://www.npmjs.com/package/html-react-parser with no code change

thanks @vladsovetov perfect 🎉

**Comment by lkraav** [comment]
> We just replaced this lib with https://www.npmjs.com/package/html-react-parser with no code change

Sounds promising, this probably also resolves #84?
